subprojects/gradle/instrumentation-tests-dex-loader/src/main/kotlin/com/avito/android/LoadTestsFromApkTask.kt

```diff
@@ -13,16 +16,16 @@ import javax.inject.Inject
 
 @CacheableTask
 @Suppress("UnstableApiUsage")
-class LoadTestsFromApkTask @Inject constructor(
+open class LoadTestsFromApkTask @Inject constructor(
```

Comment:
Gradle uses `abstract` to show an intention

Reply:
Why didn't make it abstract?

Reply:
i missed your comment, will do in next PR